homeassistant/components/imap_email_content/sensor.py

```diff
@@ -103,6 +103,8 @@ def _fetch_message(self, message_uid):
 
         if message_data is None:
             return None
+        if message_data[0] is None:
```

Comment:
Where are we checking for emtpy list? This is just checking for `None`.

What variable might be an empty list?

Reply:
Thats exactly the place. The imap library returns a list with an Object of NoneType in it. I don't know why. But it created in the following line subscript exceptions. I tried to catch that with this.

Reply:
Ok, so it's not an empty list.